pkgs/by-name/nf/nf-test/package.nix

```diff
@@ -18,14 +18,7 @@ stdenv.mkDerivation rec {
   };
   sourceRoot = ".";
 
-  buildInputs = [
-    makeWrapper
-  ];
-
-  nativeBuildInputs = [
-    nextflow
-  ];
```

Comment:
The committer who allowed this to enter should be punished! (@nbraud who merged https://github.com/NixOS/nixpkgs/pull/308756 , not @rollf ).

Reply:
> The committer who allowed this to enter should be punished! (@nbraud who merged #308756 , not @rollf ).

That sounds a bit too rude to me.

Thanks for checking/fixing this @doronbehar @SFrijters.